core/src/test/java/org/elasticsearch/monitor/fs/FsProbeTests.java

```diff
@@ -91,6 +92,33 @@ public void testFsInfo() throws IOException {
         }
     }
 
+    public void testFsInfoOverflow() throws Exception {
+        try (NodeEnvironment env = newNodeEnvironment()) {
+            FsProbe probe = new FsProbe(Settings.EMPTY, env);
+
+            FsInfo stats = probe.stats(null, null);
+            assertNotNull(stats);
+
+            FsInfo.Path total = stats.getTotal();
```

Comment:
Can you call this something other than `total`? It's really hard to read things like `total.total, total.total, total.total + total.total`.